backend/parser.py

```diff
@@ -49,7 +49,7 @@ def add_point(self, timestamp, data: list, latency=None):
         else:
```

Comment:
I can't seem to comment above this line. Please add the below text as headers for all of your files (obviously changing the comment syntax):


// Copyright 2020 Google LLC. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.